src/components/button/Button.jsx

```diff
@@ -0,0 +1,5 @@
+import "./Button.scss";
+
+export default function Button({text, type}) {

```

Comment:
Does your prettier is working on this project?

Reply:
i think so